src/en/Developer_Environment/Setting_Up.rst

```diff
@@ -0,0 +1,28 @@
+Setting up your Developer Environment
```

Comment:
Please use `git mv old_filename new_filename`, otherwise the old one is marked as deleted and new one is created from scratch without commit history, thanks!

Reply:
Good call, I'll fix this one up.

Reply:
Should be all fixed up now. I just started over, corrected the one commit with the `git mv`, skipped the rm/create step in that commit and the following one, and cherry-picked everything else back in and force-pushed it up.